### ISSUE TITLE
Running a simple experiment on the Poppy Robot

### ISSUE THREAD

**Issue by RxnDbr** [flowersteam/explauto#53]
Hi,

I am currently trying to run explauto on Poppy using your tutorial but I think it's not updated : poppy-software repository doesn't exist anymore. I think it has been replace by poppy-humanoid-beta so I cloned it and changed the config path  '../../poppy-software/poppytools/configuration/poppy_config_linux.json' by '../poppy-humanoid-beta/software/poppytools/configuration/poppy_config.json'

But when I try to run 
<pre><code>poppy = pypot.robot.from_json(config_path)</code></pre>

I got this  
<pre><code>
 File "<ipython-input-13-d81ecc662e33>", line 3, in <module>
    poppy = pypot.robot.from_json(config_path)

  File "/home/roxane/Logiciels/anaconda/lib/python2.7/site-packages/pypot/robot/config.py", line 244, in from_json
    return from_config(config, sync=sync, strict=strict, use_dummy_io=use_dummy_io)

  File "/home/roxane/Logiciels/anaconda/lib/python2.7/site-packages/pypot/robot/config.py", line 60, in from_config
    dxl_io = dxl_io_from_confignode(config, c_params, attached_ids, strict)

  File "/home/roxane/Logiciels/anaconda/lib/python2.7/site-packages/pypot/robot/config.py", line 139, in dxl_io_from_confignode
    port = pypot.dynamixel.find_port(ids, strict)

  File "/home/roxane/Logiciels/anaconda/lib/python2.7/site-packages/pypot/dynamixel/__init__.py", line 105, in find_port
    ids, list(set(ids) - set(ids_founds))))

IndexError: No suitable port found for ids [31, 32, 33, 34, 35, 36, 37, 41, 42, 43, 44, 51, 52, 53, 54]. These ids are missing [32, 33, 34, 35, 36, 37, 41, 42, 43, 44, 51, 52, 53, 54, 31] !
</code></pre>

I use spyder from anaconda to compile my script.

Do you know how to fix it ? 

Thanks,
Roxane

**Comment by pierre-rouanet** [comment]
Thanks, indeed explauto still used the old version of the poppy libraries. I'll try to fix that soon! In the meantime the patch you made should work.

The error you got is unrelated to explauto. It actually comes from the poppy library which tells you that it did not find any connected motors to your computer. Are you working with a real robot ?

**Comment by RxnDbr** [comment]
Thanks
No I don't, I am using VREP. 
Can't we use explauto with the simulator ?

**Comment by pierre-rouanet** [comment]
Yes it does. But I'm guessing you are using the "real poppy config" instead of the "v-rep one": 

```python
configurations = {'poppy': conf_poppy, 'vrep':conf_vrep, 'default':conf_vrep}
```

You can have a look at this [notebook](https://github.com/clement-moulin-frier/explaupoppypot/blob/master/results.ipynb) to see how you can do that. Hopefully we will make this step simpler in the future.

**Comment by sebastien-forestier** [comment]
@pierre-rouanet, tu pourras mettre à jour le [notebook](https://github.com/flowersteam/explauto/blob/master/notebook/poppy_environment.ipynb) qui utilise poppy-software ?
